src/main/java/mate/academy/springboot/swagger/service/mapper/DtoMapper.java

```diff
@@ -0,0 +1,7 @@
+package mate.academy.springboot.swagger.service.mapper;
+
+public interface DtoMapper<M, Q, S> {
```

Comment:
follow Interface Segregation Principle and split in into two interfaces - for request dto mapping and for response dot mapping. You can leave single class as implementation